README.md

```diff
@@ -53,7 +57,7 @@ To deploy for the first time, run:
 yarn hardhat run scripts/deployDivvi.ts --network mainnet
 ```
 
-This will initiate a proxy deployment on OpenZeppelin Defender, which requires two steps to complete - the Shared Deployer Safe wallet must sign transactions to deploy _both_ the proxy and implementation contracts. Find those steps in [Defender Dashboard](https://defender.openzeppelin.com/v2/), in the "Deploy" section for the "Celo" production environment. It deploys the implementation contract first, then the proxy one. To approve the deployment, open the deployment details and click a button to open the Safe App. If the particular Safe requires multiple signatures, ask fellow engineers for additional ones in Slack.
+This will initiate a proxy deployment on OpenZeppelin Defender, which requires two steps to complete - the Shared Deployer Safe wallet must sign transactions to deploy _both_ the proxy and implementation contracts. Note that the Shared Deployer Safe wallet is a multisig, so another team member must be available to sign the transactions. Find those steps in [Defender Dashboard](https://defender.openzeppelin.com/v2/), in the "Deploy" section for the "Ethereum" production environment. It deploys the implementation contract first, then the proxy one. To approve the deployment, open the deployment details and click a button to open the Safe App. If the particular Safe requires multiple signatures, ask fellow engineers for additional ones in Slack.
```

Comment:
Do we really need the Ethereum deployer to be 2 of X?
It used to be 1 of X.
I'm not sure we need the extra friction.

Reply:
i've removed this addition to the docs in case we want to make that change